dnf-behave-tests/fixtures/specs/repoquery-multilib/mid-a1.multilib.spec

```diff
@@ -0,0 +1,16 @@
+Name:           mid-a1
```

Comment:
Thanks for adding the repository.

So far each feature file has a corresponding repository by name, could you name it `repoquery-main-2` (since I don't think the fact the one package in it is multilib is defining for its function), or `repoquery-main-multilib` if you consider `multilib` an important part of the name? Thanks.

Reply:
As you wish, but I don't have a fun with it.